lib/ansible/modules/user.py

```diff
@@ -3114,7 +3114,12 @@ class BusyBox(User):
         - remove_user()
         - modify_user()
     """
+    def _validate_shell(self):
+        with open("/etc/shells", "rb") as f:
+            shells = [line.decode().strip() for line in f.readlines() if not line.startswith(b"#")]
 
+        if self.shell not in shells:
```

Comment:
Can we reach this point even if the shell is not provided by the user? If so we should not run this check, right?